src/pages/Landing/index.tsx

```diff
@@ -152,7 +153,7 @@ export default function Landing() {
   if (landingPageFlag === LandingPageVariant.Control || !isOpen) return null
 
   return (
-    <>
+    <Trace page="landing-page" shouldLogImpression>
```

Comment:
Should this be referred to by a constant too, i.e. `PageName.LANDING_PAGE`?

Reply:
yes